user-guide/Cloud_Platform/Catalog/About_the_Catalog_module.md

```diff
@@ -40,5 +48,15 @@ At present, the following types of Catalog items are supported:
 
 - Automation script
 - Application package
+- ChatOps Extension
 - Connector
-- Visio file
+- Data API
+- Data Grabber
+- GQI Ad-hoc Data Source
+- GQI Operator
+- Life-cycle Service Orchestration
+- Process Activitie
+- Profile-Load Script
+- Solution
+- User-defined API
+- Visio
```

Comment:
Are these the exact names of the types, or descriptions? This because there are several things I'd like to adjust (e.g. capitalization, spelling mistake in line 58).

Reply:
Woops i can indeed see the mistakes in spelling :D
This is a visually representation of the implementation in Catalog
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/138586440/401ab10e-8780-40fb-a00a-8a00d64f1697)

Reply:
ChatOps extension* small error in the image